data_collection/gazette/spiders/rj/rj_niteroi.py

```diff
@@ -9,11 +10,9 @@
 class RjNiteroiSpider(BaseGazetteSpider):
     TERRITORY_ID = "3303302"
     name = "rj_niteroi"
-    allowed_domains = ["niteroi.rj.gov.br"]
-    start_urls = ["http://www.niteroi.rj.gov.br"]
-    download_url = "http://www.niteroi.rj.gov.br/wp-content/uploads/do/{}/{}/{:02d}.pdf"
+    allowed_domains = ["niteroi.rj.gov.br", "diariooficial.niteroi.rj.gov.br"]
+    BASE_URL = "https://diariooficial.niteroi.rj.gov.br"
```

Comment:
Esse caso já está contido no outro, visto que o `diariooficial.` é subdomínio
```suggestion
    allowed_domains = ["niteroi.rj.gov.br"]
    BASE_URL = "https://diariooficial.niteroi.rj.gov.br"
```